.github/workflows/build.yml

```
@@ -10,7 +10,7 @@ on:
   push:
     branches: [ main, 'rel/*' ]
```

```suggestion
    branches: [ main, 'dev/*', 'rel/*' ]
```
I think we need it here too, but not 100% sure. that would also let us push things in 'development' to our feeds and such, but not sure if that'll have other ramifications to which feeds and that is, but probably fine for now?

---

### on: `push`
 
 This section is described in the filters docs [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#using-filters):
 > the push event has a branches filter that causes your workflow to run only when a push to a branch that matches the branches filter occurs, instead of when any push occurs.

 ```
 on:
  push:
    branches: 
 ```
 
### on: `pull_request`
 
This is described by workflow syntax docs [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore):
> When using the `pull_request` and `pull_request_target` events, you can configure a workflow to run only for pull requests that target specific branches.

  ```
 on:
  pull_request:
    branches: 
 ```

It seems like we only need to specify in `pull_request`, but we won't know for sure until we've merged this PR and rechecked in #500.

---

No dice in #500, let's try doing both then.

---

Opened https://github.com/CommunityToolkit/Windows/pull/505